### PR TITLE
Sub nav

### PR DESCRIPTION
**Pull request by puglisac**
Working on sub-nav.

Screen Shot 2021-03-02 at 4 01 16 PM

### CODE REVIEW

**Comment by daemyonc** [comment]
I modified the height of the bar a bit which you can see in new mocks in the Joblist UI issue.

**Comment by puglisac** [comment]
I modified the height and added the sign in with slack button.

Screen Shot 2021-03-03 at 10 43 41 AM

**Comment by ShaunWMusic** [comment]
547F5048-6A9B-4BAE-83DF-F3551140381E

can you line of the jobs right side with this design and also line up the slack button on the right as well?

**Comment by puglisac** [comment]
Yea that looks way nicer I think!

Screen Shot 2021-03-03 at 11 10 32 AM

File: app/styles/app.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+/* sub-nav styles */
+
+.sub-nav{
+    display: flex;
+    height: 60px;
+    background-color: #6399cc;
+    justify-content: start;
+    align-items: center;
+}
+.sub-nav-title{
```
Comment (ShaunWMusic) [inline app/styles/app.css]: we can change this file to .scss if we want to add variables that are reused

File: app/templates/application.hbs [hunk 2]
```diff
@@ -1 +1,5 @@
-{{outlet}}
\ No newline at end of file
+<TopNav />
+<SubNav />
+{{!-- <main>
```
Comment (ShaunWMusic) [inline app/templates/application.hbs]: we can remove the commented out section

File: tests/integration/components/sub-nav/component-test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | sub-nav', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
```
Comment (ShaunWMusic) [inline tests/integration/components/sub-nav/component-test.js]: feel free to add tests and try it out

**Review by ShaunWMusic — APPROVED** [review]
looking good! once damion merges his, then you can merge this one

File: tests/integration/components/sub-nav/component-test.js (hunk 3 above)
Comment (puglisac) [reply tests/integration/components/sub-nav/component-test.js]: Thank you Shaun! I added a couple tests.

**Review by daemyonc — APPROVED** [review]

**Pull request merged**